SDK/CalendarStoreController.swift

```diff
@@ -0,0 +1,148 @@
+    let pageIdentifier: String?
+    
+    /// The title to be used on the home page's navigation bar.
+    let homePageTitle: String?
+    
+    /// The global tint color used through out the SDK.
+    let tintColor: UIColor
+    
+    /** **For iOS 11.0+**
+     Set to false if you don't want to use large navigation bar titles.
+    */
+    let largeTitle: Bool
+    
+    // - MARK: Initialization
+    
+    // This method is only called when initializing a `UIViewController` from a `Storyboard` or `XIB`. The `CalendarStoreController` must only be used programatically, but every subclass of `UIViewController` must implement `init?(coder aDecoder: NSCoder)`.
+    public required init?(coder aDecoder: NSCoder) {
+        fatalError("CalendarStoreController must only be initialized programatically.")
+    }
+    
+    /**
+     - parameter apiKey: The API Key (access token) for the **SchedJoules API**.
+     - parameter pageIdentifier: The page identifier for the the home page.
+     - parameter title: The title for the `navigtaion bar` in the home page.
+     - parameter largeTitle: Set to `false` if you don't want to use large navigation bar titles.
+     - parameter tintColor: The tint color used through out the SDK, default is SchedJoules red.
+     */
+    public init(apiKey: String, pageIdentifier: String?, title: String?, largeTitle: Bool = true, tintColor: UIColor = ColorPalette.red) {
+        // Initialization
+        self.apiKey = apiKey
+        self.pageIdentifier = pageIdentifier
+        self.largeTitle = largeTitle
+        self.tintColor = tintColor
+        homePageTitle = title
+        super.init(nibName: nil, bundle: nil)
+        
+        // Set tab bar tint color
+        tabBar.tintColor = tintColor
+
+        // Array to hold the view controllers
+        var tabViewControllers = [UIViewController]()
+    
+        // Create home page with a specific page identifier
+        if pageIdentifier != nil {
+            let homeVC = PageViewController.init(apiKey: apiKey, pageQuery: SinglePageQuery(pageID: pageIdentifier!, locale: readSettings().last!), searchEnabled: true)
+            homeVC.title = title
+            tabViewControllers.append(homeVC)
+        // Create home page with juts localization parameters
+        } else {
+            let homeVC = PageViewController(apiKey: apiKey, pageQuery: HomePageQuery(locale: readSettings().first!, location: readSettings().last!), searchEnabled: true)
+            homeVC.title = title
+            tabViewControllers.append(homeVC)
+        }
+        
+        // Create settings page
+        let storyBoard = UIStoryboard.init(name: "SDK", bundle: nil)
+        let settingsVC = storyBoard.instantiateViewController(withIdentifier: "SettingsViewController") as! SettingsViewController
+        settingsVC.accessToken = apiKey
+        settingsVC.title = "Settings"
+        tabViewControllers.append(contentsOf: [homeVC,settingsVC])
```

Comment:
Doesn't compile for me. I'm getting an "Use of unresolved identifier 'homeVC'" at this line.